### ISSUE TITLE
Idea: press shift to show right click menu option

### ISSUE THREAD

**Issue by ObliqueBoredom** [donth77/loot-lookup-plugin#13]
Don't know if its possible, but would be a nice feature to have the option for the menu to only show up while holding shift (or a similar key). would be nice to de clutter right click menu.

**Comment by donth77** [comment]
I'm not sure if it would be possible to dynamically add and remove options while the menu is open. According to the RuneLite Client API docs, [setMenuEntries](https://static.runelite.net/api/runelite-api/net/runelite/api/Client.html#setMenuEntries(net.runelite.api.MenuEntry[])) should typically be used in the context of the [MenuOpened](https://static.runelite.net/api/runelite-api/net/runelite/api/events/MenuOpened.html) event.

**Comment by donth77** [comment]
@ObliqueBoredom 
I'm not sure if it would be possible to dynamically add and remove options while the menu is open. According to the RuneLite Client API docs, [setMenuEntries](https://static.runelite.net/api/runelite-api/net/runelite/api/Client.html#setMenuEntries(net.runelite.api.MenuEntry[])) should typically be used in the context of the [MenuOpened](https://static.runelite.net/api/runelite-api/net/runelite/api/events/MenuOpened.html) event.

**Comment by CalebBarnes** [comment]
Maybe possible to check if the key is currently held down when the user right-clicks instead of dynamically adding/removing from the menu?

**Comment by ObliqueBoredom** [comment]
Yeah, Caleb pretty much nailed it, just want the option not to be there in normal use, but show up in the menu if I'm pressing a certain key before/during right clicking. Also thanks for hearing me out.

p.s. also remembered the ground markers plugin does the function i was asking. while holding shift click the "mark tile" option shows up on ground right clicks.

**Comment by Daleyy** [comment]
Not having this option also drove me nuts, so I opened a PR to implement this change: https://github.com/donth77/loot-lookup-plugin/pull/51
